Updated echo to r1371

dc-redeclipse opened the pull request:
- some texture and geo fixes
- add some decals
- improved clip material on some places

dc-redeclipse commented:
I dont know why always these conflicts happend but this zip file is the right content for the merge.
[echo-r1371.zip](https://github.com/redeclipse/maps/files/5421625/echo-r1371.zip)

dc-redeclipse commented:
just ignore the conflict the newest version is in the zip file just to get it right

Jigoku commented:
Did you make sure your local maps repo is in sync with previous changes?
Maybe check with:

```
git checkout dc-de:echo-r1371
git pull upstream master
git reset --hard upstream/master
```
Then readd/commit your changes for the updated map and;
```
git push origin dc-de:echo-r1371 --force
```

That should probably clear up the conflicts.

dc-redeclipse committed:
Updated echo to r1371

- some texture and geo fixes
- add some decals
- improved clip material on some places
